clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -127,6 +127,15 @@ public synchronized void addTopics(String... topics) {
     }
 
     /**
+     * Add one or more topics to maintain metadata for
+     */
+    public synchronized void addTopics(Iterable<String> topics) {
```

Comment:
Just out of curiosity: any special reason to use an ``Iterable`` instead of a ``Collection``?